snap/snapcraft.yaml

```diff
@@ -41,11 +41,27 @@ parts:
     source-type: git
     plugin: go
     go-channel: latest/stable
-        
+
```

Comment:
Line endings?

Reply:
Oh whoops. Thanks.

Reply:
Oh I guess that was already with trailing whitespace--does snapcraft need that when lines are spaced like that?

Reply:
No idea, I didn't set this up. I probably missed this during the review. Guessing it doesn't matter though, so feel free to keep your changes.

Reply:
It's fine either way.